xla/service/latency_hiding_scheduler.cc

```diff
@@ -1902,9 +1908,21 @@ absl::StatusOr<HloGraphNode::TimeCost> DefaultSchedulerCore::ScheduleNode(
   ++sched_state->scheduled_count;
   for (auto& resource : n->GetResources()) {
     if (resource.second == ResourceUsageType::kResourceRelease) {
-      --sched_state->resources_in_flight[resource.first];
+      sched_state->resource_occupiers_in_flight.at(resource.first)
```

Comment:
This breaks `LatencyHidingSchedulerTest.AllGatherWithSelectiveOverlap`.

Reply:
thanks! fixed